### PR TITLE
Expose an API to set the database filename. r=francisco

### PR DESCRIPTION
**Pull request by ferjm**

### CODE REVIEW

File: src/auth_middleware.rs [hunk 1]
```diff
@@ -263,7 +263,7 @@ describe! auth_middleware_tests {
         use crypto::sha2::Sha256;
         use jwt;
 
-        let db = UsersDb::new();
+        let db = UsersDb::new(Some("./users_db.sqlite".to_owned()));
```
Comment (arcturus) [inline src/auth_middleware.rs:266]: Do we need to specify the db here?

We could just do

``` rust
let db = UsersDb::new(None);
```

File: src/users_db.rs [hunk 2]
```diff
@@ -462,7 +458,7 @@ describe! user_builder_tests {
 #[cfg(test)]
 describe! user_db_tests {
     before_each {
-        let usersDb = UsersDb::new();
+        let usersDb = UsersDb::new(Some("./users_db.sqlite".to_owned()));
```
Comment (arcturus) [inline src/users_db.rs:461]: Should we use perhaps a different db for the tests?

``` rust
let usersDb = UsersDb::new(Some("./test_users_db.sqlite".to_owned()));
```

File: src/users_router.rs [hunk 3]
```diff
@@ -507,7 +507,7 @@ describe! login_tests {
         }
 
         let router = UsersRouter::init();
-        let usersDb = UsersDb::new();
+        let usersDb = UsersDb::new(Some("./users_db.sqlite".to_owned()));
```
Comment (arcturus) [inline src/users_router.rs:510]: Same comment than above, if it's the default db we could use None

**Comment by arcturus** [comment]
Looking good to me, left a couple of messages, but they are not important.

@ferjm please, rebase and merge!

**Comment by ferjm** [comment]
It seems that #66 superseded this PR

**Pull request closed without merging**